cdc/server.go

```diff
@@ -166,8 +170,10 @@ func (s *Server) Run(ctx context.Context) error {
 
 // startStatusHTTP starts the HTTP server.
 // `lis` is a listener that gives us plain-text HTTP requests.
-// TODO can we decouple the HTTP server from the capture server?
+// TODO: can we decouple the HTTP server from the capture server?
 func (s *Server) startStatusHTTP(lis net.Listener) error {
+	// limit the max concurrent conections of this statusServer.
+	lis = netutil.LimitListener(lis, maxHTTPConnection)
```

Comment:
What happens when it reaches limit?

Reply:
The TCP connections are queued up and no new goroutines will be created until a connection is processed.

Reply:
Are queued connections affected by `ReadTimeout` and `WriteTimeout`? Does TiCDC close connections actively if they are timeout?

Reply:
Can TiCDC log closed connections?